packages/common/container-definitions/src/loader.ts

```diff
@@ -570,7 +570,9 @@ export interface IHostLoader extends ILoader {
 }
 
 /**
- * @alpha
+ * @privateRemarks
+ * used by fluidframework/datastore-definitions public IFluidDataStoreRuntime
+ * @public
```

Comment:
In the past we have had some debate about what is called "loader" and what should be called "loader" (see https://github.com/microsoft/FluidFramework/pull/14003 ). If we are going to make anything referring to loader public, we should sort that out, and to any renames and clarifications needed before stabalizing.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6594